package.json

```diff
@@ -15,13 +15,9 @@
   "homepage": "https://github.com/frinyvonnick/gitmoji-changelog#readme",
   "scripts": {
     "test": "jest",
-    "lint": "eslint packages/**/*.js",
-    "build": "lerna run build --stream"
+    "lint": "eslint packages/**/*.js"
```

Comment:
I am more : `    "lint": "eslint packages/**/src"` in my lerna repository since I put all my code into `src` sub directories.

It allows me to have `mjs`, `jsx` extensions if I want to (this is `--ext` parameter of eslint, so you should not specify it here), and it allows me to don't have a `eslintignore` rule